### ISSUE TITLE
Feature request: load queue from playlist file (.m3u)

### ISSUE THREAD

**Issue by ericsfraga** [alphapapa/listen.el#7]
I have a large number of playlist files (.m3u extension). listen is able to play such files. I would also, however, like to create a listen queue by adding tracks from such files.

Otherwise, I like the simplicity of this package!

**Comment by alphapapa** [comment]
Yeah, that's a great idea.  I have playlist files like that too.  Thanks for the suggestion.

**Comment by ericsfraga** [comment]
Excellent. Thank you. I will try it when 0.3 lands in ELPA!

**Comment by ericsfraga** [comment]
Just installed v0.4. Really nice! and lower case transient commands which is a big bonus (for those of us with RSI...). Thanks again.

**Comment by alphapapa** [comment]
Thanks, glad you like it.  And yeah, I've been tweaking the bindings.  Feedback is always welcome in that regard.
